SparkPost/support-docs
# WE-313: mobile header

### What Changed
- makes the header responsive with a hamburger menu
- makes the mobile menu

### How to test
- run locally or try the preview link
- verify that the mobile nav works in the drawer as expected
- verify the desktop left nav hides at mobile widths
